### PR TITLE
Implemented Emojis Package and made default comments more complex

### PR DESCRIPTION
**Pull request by eliareutlinger**
Since I had errors with the current format of emojis in the comments I looked for another way to solve this. My solution is the [emoji package](https://pypi.python.org/pypi/emoji/). You can see the cheat-sheet [here](https://www.webpagefx.com/tools/emoji-cheat-sheet/).

I updated the default comments to fit the package and made them more complex, since I noticed that you get blocked quite fast if you post the same comments twice.

Would be greate if someone could check & test my changes first because i'm still a beginner in python :)

EDIT:
The comments can't be logged anymore, since the logger seems to missunderstand the format of the emojis. I weren't able to fix that yet :(
image

### CODE REVIEW

**Comment by nickpettican** [comment]
Looks good @eliareutlinger , have you checked the emojis appear correctly on the Instagram comments?
We should also change the README to reflect the changes. I'll test it out when I get some free time this week :), and also fix the logging issue.

**Comment by eliareutlinger** [comment]
@nickpettican jep, they appear correctly:
image
I know the complexity makes it partly ugly but i'm runnig it now since one hour and didn't got blocked yet :)

**Comment by eliareutlinger** [comment]
Well, not all emojis from the given cheat-sheet seem to work.
For Example ":simple_smile:" wasn't even translated by the emoji-pack. It just says :simple_smile: in the comment. So I replaced it with :smiley: which is working properly. 

So idk which ones are actually working fine, but the ones in the default config do. Seems like you'd have to check others first before using them.

**Comment by eliareutlinger** [comment]
I think I fixed the Loggin now (by printing the emojis in their alphabetic code first) and added some more comments. I think this should be working but since Instagram seems to have a big problem with their Login-UI I weren't able to test it yet 👎

**Comment by eliareutlinger** [comment]
The emojis are now working fine. I tested everything this day for about 8 Hours on Raspbian and had no single Error logged. Additionally I changed the console output/logging structure to display informations clearer.

File: config.json [hunk 1]
```diff
@@ -11,10 +11,61 @@
 	"follow_in_day": 50,
 	"unfollow": true,
 	"follow_time_hours": 6,
-	"comments_in_day": 0,
-	"comments_list": [["Cool", "Sweet", "Awesome", "Great"],
-						["😄", "🙌", "👍", "👌", "😊"], 
-						[".", "!", "!!", "!!!"]],
+	"comments_in_day": 50,
```
Comment (nickpettican) [inline config.json]: Good work here

File: src/instabot.py [hunk 2]
```diff
@@ -300,15 +301,15 @@ def catch_up_operations(self):
...
+
         if self.banned['400'] < 3:
             sleep_time = 5*60
 
         else:
             sleep_time = 60*60
             self.banned['banned'] = True
             self.banned['400'] = 0
-        self.console.log('\tSleeping for %s minutes... '%(sleep_time/60))
+        self.console.log('\n\t--- Sleeping for %s minutes ---'%(sleep_time/60))
```
Comment (nickpettican) [inline src/instabot.py]: Better logging indeed

File: src/instabot.py [hunk 3]
```diff
@@ -685,31 +687,28 @@ def unfollow_user(self):
...
-                self.console.log('\n * Trying to unfollow %s... \,' %(username))
+                self.console.log('\n * Trying to unfollow "%s": \,' %(username))
                 self.profile.remove_follow(user_id)
                 return self.explore_operation('unfollow', user)
         return [True]
 
     def user_following_back(self, username):
         # check if the user is following back
 
-        user_data = check_user(self.browser, self.insta_urls['user'], username)
```
Comment (nickpettican) [inline src/instabot.py]: Why was this removed?

File: src/instabot.py (hunk 2 above)
Comment (eliareutlinger) [reply src/instabot.py]: Not better, but clearer in console (for me at least)

**Comment by eliareutlinger** [comment]
I removed the function for betterlocal testing (on my windows machine), since I didn't want to follow people with a test account which is never active. 

Then I forgot to redo it, but here you go

File: src/instabot.py [hunk 4]
```diff
@@ -754,8 +756,9 @@ def explore_operation(self, operation, identifier):
             comment = False
             # comment
             if operation == 'comment':
-                comment = generate_comment(self.params['comments_list'])
-                self.console.log(comment + '\,')
+                commentString = generate_comment(self.params['comments_list'])
+                self.console.log('\n\n\tComment: "' + commentString + '" \n')
```
Comment (nickpettican) [inline src/instabot.py]: Should we remove the last `\n` and add `\,` to have `Success` printed in the same line? Otherwise it looks like this:
```
[ 2018-03-27 16:11:55 ]  * Trying to comment "https://www.instagram.com/p/":  

	Comment: "Cool :star: :clap: Keep it up!" 

Success
```

Comment (nickpettican) [reply src/instabot.py]: Other than that it works fine 👍 I'll merge once you reply to this ☝️ @eliareutlinger

Comment (eliareutlinger) [reply src/instabot.py]: I didn't really knew where to should put that `Success` because of the long comments. But yes that'd look better.
